js/wrlc-announce.module.js

```diff
@@ -1,44 +1,47 @@
-angular
-  // Define the module name
-  .module('wrlcAnnounce', ['ngAnimate'])
-    .component('prmSearchBarAfter', {
-        template: `
-            <wrlc-announce ng-show="!$ctrl.dismissed" ng-if="$ctrl.show">
-                <div id="wrlc-announce-banner" class="layout-align-center-center layout-row flex">
-                    <prm-icon icon-type="svg" svg-icon-set="action" icon-definition="ic_announcement_24px" id="wrlc-announce-icon"></prm-icon>
-                    <span ng-if="$ctrl.link" id="message"><a href="{{$ctrl.link}}">{{ $ctrl.message }}</a></span>
-	            <span ng-if="!$ctrl.link" ng-bind-html=$ctrl.message id="message"></span>
-	        <button id="dismiss-announcement" area-label="dismiss announcement" class="dismiss-alert-button zero-margin md-button md-primoExplore-theme md-ink-ripple button-with-icon" type="button" ng-click="$ctrl.wrDismiss()">
-	        	<prm-icon icon-type="svg" svg-icon-set="navigation" icon-definition="ic_close_24px" class="material-icons gray"></prm-icon>
-		</button>
-                </div>
-            </wrlc-announce>
-            `,
-        controller: 
-            function announceController(announceConfig, $http) {
-	    
+// specify module name and its dependency
+angular.module('wrlcAnnounce', ['ngAnimate'])
 
-                var self = this;
-                var config = announceConfig;
+.component('prmSearchBarAfter', {
+  template: `
+    <wrlc-announce ng-show="!$ctrl.dismissed" ng-if="$ctrl.show">
+      <div id="wrlc-announce-banner" class="layout-align-center-center layout-row flex {{$ctrl.severity}}">
+        <prm-icon ng-if="$ctrl.severity=='info'" icon-type="svg" svg-icon-set="action" icon-definition="ic_info_24px"></prm-icon>
+        <prm-icon ng-if="$ctrl.severity=='alert'" icon-type="svg" svg-icon-set="action" icon-definition="ic_announcement_24px"></prm-icon>
+        <prm-icon ng-if="$ctrl.severity=='warn'" icon-type="svg" svg-icon-set="action" icon-definition="ic_report_problem_24px"></prm-icon>
+        <prm-icon ng-if="$ctrl.severity=='success'" icon-type="svg" svg-icon-set="action" icon-definition="ic_check_circle_24px"></prm-icon>
+        <span ng-if="$ctrl.link" id="message"><a target="_blank" href="{{$ctrl.link}}">{{ $ctrl.message }}</a></span>
```

Comment:
just a quick note here that i did add the `target="_blank"` here to open the link in a new tab. I can remove this or add another customization variable to make sure nobody sees any unexpected behavior.